### DISCUSSION TITLE
ChromeOS Support?

### DISCUSSION THREAD

**Discussion by Anks329** [cisagov/ScubaGoggles#244] [Q&A]
Does the ScubaGoggles tool support ChromeOS events in the audit log as well to provide a baseline for organizations using ChromeOS devices?

**Comment by buidav** [comment]
No. 
ScubaGoggles does not currently support ChromeOS events nor do we have a security baseline for ChromeOS devices.
The [SCuBA project's](https://cisa.gov/scuba) scope is currently focused on securing cloud services. 
Thus, endpoint management is out of scope of the program.
